src/PresentationalComponents/Breadcrumbs/Breadcrumbs.js

```diff
@@ -66,7 +69,7 @@ const Breadcrumbs = ({ current }) => {
           <BreadcrumbItem isActive>{current}</BreadcrumbItem>
         </Breadcrumb>
       ) : (
-        intl.formatMessage(messages.loading)
+        messages.loading.defaultMessage
```

Comment:
We are making a soft deprecation of intl. This means, you no longer need to use the package and can freely insert a plain string "Loading..." here instead.

Reply:
Isn't it what I am doing here ? 😅 
I removed the intl line and inserted a use of `messages` object, which inserts a raw string 

This is the `loading` attribute of messages:
![loading](https://github.com/RedHatInsights/insights-advisor-frontend/assets/93318917/4da31a2d-3ca1-475f-b876-c646aff61f75)

Reply:
I mean, you don't even need to use `messages` because it's already a part of react-intl API (see what is exported in Messages.js) 🙂

Reply:
I can see what you mean, but I thought we were trying to avoid using react-intl

@gkarat please have a look at this comment:
https://github.com/RedHatInsights/insights-advisor-frontend/pull/1164#issuecomment-1782566517 
That was the reason behind this line

Just wanted to make sure we are all on the same page here before updating this line @gkarat @Fewwy

Reply:
My whole point is about using `"Loading..."` here directly instead of `messages.loading.defaultMessage`:
```suggestion
        Loading...
```

In my previous comment, I was trying to point out that if you use the `messages` variable from the Messages.js module, you utilise intl automatically. We want to avoid this, and this is why I suggest to replace with just a plain text (messages.loading.defaultMessage is not).